ps_customeraccountlinks.tpl

```diff
@@ -24,11 +24,33 @@
  *}
 
 <div id="block_myaccount_infos">
-	<h4><a href="{$urls.pages.my_account}" title="{l s='Your account' d='Shop.Theme.Customeraccount'}" rel="nofollow">{l s='Your account' d='Shop.Theme.Customeraccount'}</a></h4>
-	<ul>
-    {foreach from=$my_account_urls item=my_account_url}
-        <li><a href="{$my_account_url.url}" title="{$my_account_url.title}" rel="nofollow">{$my_account_url.title}</a></li>
-    {/foreach}
-    {hook h="displayMyAccountBlock"}
-	</ul>
+  <h4><a href="{$urls.pages.my_account}" title="{l s='Your account' d='Shop.Theme.Customeraccount'}" rel="nofollow">{l s='Your account' d='Shop.Theme.Customeraccount'}</a></h4>
```

Comment:
Using my_account_urls allows people to add more value from hooks. Not sure putting all links here is the right solution

Reply:
@PierreRambaud They should use {hook h="displayMyAccountBlock"} for that, and not overriding the module. (how would that be possible? :-))

Reply:
Everything is possible with PrestaShop you know ;)

Reply:
@PierreRambaud We have all the URLs generated from the core, I don't see why we should keep some legacy code and slow down the development because of wrong practices. 😛

They have a nice hook that every module uses. ;-)